common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -2370,7 +2370,7 @@ def test_solutions_not_indexed(self):
     def test_indexing_checkboxes(self):
         name = "Checkboxes"
         descriptor = self._create_descriptor(self.sample_checkbox_problem_xml, name=name)
-        capa_content = textwrap.dedent("""
+        capa_content = textwrap.dedent(u"""
```

Comment:
This string contains literal emdashes.  In python 3, including unicode characters in bytestrings is an error.  Here, it causes an error when `.decode('ascii')`ing capa_content in the assertEquals below.